docs/index.md

```diff
@@ -11,4 +11,7 @@ MLXPRS is an extension for Visual Studio Code that simplifies and speeds up the
 * Real-time query evaluation of JavaScript, XQuery, SQL, SPARQL, Optic, and GraphQL against a MarkLogic instance
 * View modules (read-only) in the editor
 * Run marklogic-unit-test modules
-* Validate TDE templates and test the templates with node extraction
\ No newline at end of file
+* Validate TDE templates and test the templates with node extraction
+* View the databases and app-servers deployed to the currently configured MarkLogic server
```

Comment:
You can keep the hyphen, but "app servers" doesn't require one. Also, end each of these bullet points with a period - I know the docs team prefers that, as I used to omit a period unless it was a real sentence.